### PR TITLE
including surface wind speed speed with quivers into the weather briefing 

### PR DESCRIPTION
**Pull request by dspraturi**
These code changes should close the issue #16. Here is a sample figure that gets generated using this function: 
test

### CODE REVIEW

File: src/wblib/figures/internal/sfc_wind_quivers.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                bbox = dict(facecolor='white',
+                            edgecolor='none',
+                            alpha=1))
+    
+# # if name == "main":
+# briefing_time = pd.Timestamp(2024, 8, 4, tz="UTC")
+# lead_hours = "048H"
+# fig = surface_wind_quivers(briefing_time, lead_hours)
+# # fig.show()
+# fig.savefig("test.png")
```
Comment (lkluft) [inline src/wblib/figures/internal/sfc_wind_quivers.py:107]: I would remove lines that are not in use. They tend to get outdated quite fast and will likely not work if uncommented in future.

File: src/wblib/services/_define_figures.py [hunk 2]
```diff
@@ -17,6 +18,7 @@
 }
 
 INTERNAL_PLOTS = {
-    "iwv_itcz_edges": iwv_itcz_edges
+    "iwv_itcz_edges": iwv_itcz_edges,
+    "surface_wind_quivers": surface_wind_quivers
```
[the remark this answers is not in the document]
Comment (dspraturi) [reply src/wblib/services/_define_figures.py]: done 👍

File: src/wblib/figures/internal/sfc_wind_quivers.py (hunk 1 above)
Comment (dspraturi) [reply src/wblib/figures/internal/sfc_wind_quivers.py]: done 👍

**Review by lkluft — APPROVED** [review]
For me this looks fine 👌 Likely @sortega87 and @HF7weatherman know better about the internal style guide of your repo 😉

**Review by HF7weatherman — APPROVED** [review]

File: src/wblib/services/_define_figures.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 from wblib.figures.external.noaa import nhc_seven_days_outlook
 from wblib.figures.external.noaa import nhc_surface_analysis_atlantic
 from wblib.figures.internal.icwv import iwv_itcz_edges
+from wblib.figures.internal.sfc_wind_quivers import surface_wind_quivers
```
[the remark this answers is not in the document]
Comment (dspraturi) [reply src/wblib/services/_define_figures.py]: done! 👍

File: src/wblib/services/_define_figures.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (dspraturi) [reply src/wblib/services/_define_figures.py]: done! 👍

File: src/wblib/figures/internal/sfc_wind_quivers.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+SPEED_THRESHOLD = 3  # m/s
+SPEED_MAX = 15  # m/s
+SPEED_MIN = 0  # m/s
+SPEED_COLORMAP = "YlGn"
+FIGURE_SIZE = (15, 8)
+FIGURE_BOUNDARIES = (-65, -5, -10, 20)
+MESH_GRID_SIZE = 50
+QUIVER_SKIP = 4
+
+def surface_wind_quivers(briefing_time: pd.Timestamp, lead_hours: str) -> Figure:
```
[the remark this answers is not in the document]
Comment (dspraturi) [reply src/wblib/figures/internal/sfc_wind_quivers.py]: done 👍

**Pull request merged**